sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/MetricsAdvisorClient.cs

```diff
@@ -176,630 +123,292 @@ public virtual Pageable<string> GetMetricDimensionValues(string metricId, string
+        /// <exception cref="ArgumentNullException"><paramref name="metricId"/> is null.</exception>
         /// <exception cref="ArgumentException"><paramref name="metricId"/> is empty or not a valid GUID.</exception>
-        public virtual AsyncPageable<MetricSeriesDefinition> GetMetricSeriesDefinitionsAsync(string metricId, GetMetricSeriesDefinitionsOptions options, CancellationToken cancellationToken = default)
+        [ForwardsClientCalls]
+        public virtual AsyncPageable<MetricFeedback> GetAllFeedbackAsync(string metricId, GetAllFeedbackOptions options = default, CancellationToken cancellationToken = default)
         {
-            Argument.AssertNotNull(options, nameof(options));
-
             Guid metricGuid = ClientCommon.ValidateGuid(metricId, nameof(metricId));
-            MetricSeriesQueryOptions queryOptions = new MetricSeriesQueryOptions(ClientCommon.NormalizeDateTimeOffset(options.ActiveSince));
-
-            int? skip = options.Skip;
-            int? maxPageSize = options.MaxPageSize;
+            FeedbackFilter filter = options?.Filter;
 
-            // Deep copy filter contents from options to queryOptions.
+            MetricFeedbackFilter queryOptions = new MetricFeedbackFilter(metricGuid)
+            {
+                DimensionFilter = filter
+            };
 
-            foreach (KeyValuePair<string, IList<string>> kvp in options.DimensionCombinationsFilter)
+            if (filter != null)
             {
-                queryOptions.DimensionFilter.Add(kvp.Key, new List<string>(kvp.Value));
+                queryOptions.EndTime = filter.EndsOn;
+                queryOptions.FeedbackType = filter.FeedbackKind;
+                queryOptions.StartTime = filter.StartsOn;
+                queryOptions.TimeMode = filter.TimeMode;
             }
 
-            async Task<Page<MetricSeriesDefinition>> FirstPageFunc(int? pageSizeHint)
+            int? skip = options?.Skip;
+            int? maxPageSize = options?.MaxPageSize;
+            RequestContext context = new RequestContext()
             {
-                using DiagnosticScope scope = _clientDiagnostics.CreateScope($"{nameof(MetricsAdvisorClient)}.{nameof(GetMetricSeriesDefinitions)}");
-                scope.Start();
+                CancellationToken = cancellationToken,
+            };
 
-                try
-                {
-                    Response<MetricSeriesList> response = await _serviceRestClient.GetMetricSeriesAsync(metricGuid, queryOptions, skip, maxPageSize, cancellationToken).ConfigureAwait(false);
-                    return Page.FromValues(response.Value.Value, response.Value.NextLink, response.GetRawResponse());
-                }
-                catch (Exception e)
-                {
-                    scope.Failed(e);
-                    throw;
-                }
-            }
+            using DiagnosticScope scope = ClientDiagnostics.CreateScope($"{nameof(MetricsAdvisorClient)}.{nameof(GetAllFeedback)}");
```

Comment:
IIRC, we don't want this here b/c everything contained in this scope is a delegate.  How are we validating distributed tracing scopes for this experiment?

Reply:
For this paging method,  convenience method name is different than DPG method that's why I added the scope here.  We don't want this scope here?

Currently there is no validation added for distributed tracing scopes for this experiment. Do you know if other languages are validating distributed tracing scopes?

Reply:
Removed outer scope for paging methods.